crates/api_models/src/payouts.rs

```diff
@@ -1 +1,434 @@
+#[cfg(feature = "payouts")]
```

Comment:
You could have moved the entire `payouts` module behind the feature flag, in `api_models/src/lib.rs` instead of adding it on every import and type here.